R/api.R

```diff
@@ -442,6 +450,61 @@ nv_reduce_any <- nvl_reduce_any
 #' @export
 nv_reduce_all <- nvl_reduce_all
 
+#' @title Random Numbers
+#' @name nv_rng_bit_generator
+#' @description
+#' generate random bits of desired shape and dtype
+#' @param initial_state state seed
+#' @param rng_algorithm one of 'DEFAULT', 'THREE_FRY', 'PHILOX'
+#' @param dtype datatype of output
+#' @param shape_out output shape
+#' @export
+nv_rng_bit_generator <- nvl_rng_bit_generator
+
+#' @title Random Uniform Numbers
+#' @name nv_runif
+#' @description
+#' generate random uniform numbers
+#' @param initial_state state seed
+#' @param shape_out output shape
+#' @param lower lower bound
+#' @param upper upper bound
+#' @export
+nv_runif <- function(initial_state, shape_out, lower = 0, upper = 1) {
```

Comment:
Support different float output types: `"f32"` and `"f64"` via a `type` argument.